actions/fgCopyActionHelper.js

```diff
@@ -54,12 +52,13 @@ class FloodgateActionHelper {
                             // Save the file in the floodgate destination location
                             const saveStatus = await sharepoint.saveFile(file, destination, true);
```

Comment:
@raga-adbe-gh This is calling the saveFile() which still has the delete file on locked code as I see in `sharepoint.js`. So for copy from source to pink, it is still overwriting but for promote it is creating a new version?

Reply:
Thanks @sukamat, Correct for copy flow is unchanged. The content write (new version) is only added for promote when content is moved from pink tree to main tree.